clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10383,6 +10382,7 @@ void SPIRVTranslator::ConstructJob(Compilation &C, const JobAction &JA,
     if (C.getDriver().IsFPGAHWMode())
       // Enable several extensions on FPGA H/W exclusively
       ExtArg += ",+SPV_INTEL_usm_storage_classes,+SPV_INTEL_runtime_aligned"
+                ",+SPV_INTEL_arbitrary_precision_integers"
```

Comment:
The description states `FPGA` and does not scrutinize between hardware or emulation.  Should this also apply to emulation?

Reply:
@MrSidims Should this also apply to emulation?

Reply:
Yes, the extension should be enabled for both hardware and emulation in AOT compilation.

Reply:
Updated to add SPV_INTEL_arbitrary_precision_integers for FPGA emulation targets.